lib/mbr.ml

```diff
@@ -296,3 +296,42 @@ let marshal (buf : Cstruct.t) t =
 
 let sizeof = sizeof_mbr
 let default_partition_start = 2048l
+
+let print_mbr_fields mbr =
+  Printf.printf "MBR fields:\n";
+  Printf.printf "  bootstrap_code: %s\n" mbr.bootstrap_code;
```

Comment:
`mbr.bootstrap_code` contains binary (executable) data. In OCaml you can use the `%S` printf modifier to print it as an escaped string, but even better would be to print it as hexadecimal.

Reply:
Thank you. I will see how to implement the `cmdliner` library. It looks great with good documentation.

Reply:
Sounds good! Please go ahead and open an issue.